.resinci.yml

```diff
@@ -0,0 +1 @@
+reviewers: 0
```

Comment:
0 required reviewers?

Reply:
@CameronDiver should not be the case? should always have reviewers even for playground projects?

Reply:
Ah sorry, didn't look at the org that we were in :)